.appveyor.yml

```diff
@@ -138,6 +138,7 @@ before_build:
   - set PATH=%PATH:C:\Program Files\Git\usr\bin;=%
   - set PATH=C:\MinGW\bin;%PATH%
   - if /I "%GENERATOR%" == "MinGW Makefiles" (set "GENERATOR_ARGS=") else (set "GENERATOR_ARGS=-A %PLATFORM%")
+  - if /I "%GENERATOR%" == "MinGW Makefiles" (set "INSTALL_TARGET=install") else (set "INSTALL_TARGET=INSTALL")
```

Comment:
Are you sure that lowercase `install` does not work with MSVC?

Reply:
lowercase `install` is working with MSVC, see: https://ci.appveyor.com/project/dlfcn-win32/dlfcn-win32/builds/22271048/job/p12xqd0frrr7oq0q#L159

Reply:
Cool, great.

Reply:
Fixed in https://github.com/dlfcn-win32/dlfcn-win32/pull/48/commits/56c6cbc1bb7d2d103bcec2251b54b4d69292b654 .